code/game/objects/items/weapons/twohanded.dm

```diff
@@ -546,9 +541,7 @@ obj/item/weapon/twohanded/
 /obj/item/weapon/twohanded/knighthammer
 	name = "singuloth knight's hammer"
 	desc = "A hammer made of sturdy metal with a golden skull adorned with wings on either side of the head. <br>This weapon causes devastating damage to those it hits due to a power field sustained by a mini-singularity inside of the hammer."
-
-	icon_override = 'icons/mob/in-hand/swords.dmi'
-	icon_state = "adrhammer0"
+	icon_state = "knighthammer0"

```

Comment:
It's the only weapon in the game with a reference to one of Paradise's players, which is not something we want to do or encourage.